demos/interactive_face_detection_demo/face.cpp

```diff
@@ -17,17 +17,17 @@ Face::Face(size_t id, cv::Rect& location):
 }
 
 void Face::updateAge(float value) {
-    _age = (_age == -1) ? value : 0.95f * _age + 0.05f * value;
+    _age = value;
 }
 
 void Face::updateGender(float value) {
     if (value < 0)
         return;
 
     if (value > 0.5) {
-        _maleScore += value - 0.5f;
+        _femaleScore += value - 0.5f;
     } else {
-        _femaleScore += 0.5f - value;
+        _maleScore += 0.5f - value;
```

Comment:
Why it has changed?

Reply:
It was not necessary.
Returned

Reply:
Do you understand what the demo app is doing here?

Reply:
Can you explain it to me?